scripts/aws-config.json

```
@@ -0,0 +1,77 @@
+{
+  "IndexDocument": {
+    "Suffix": "index.html"
+  },
+  "ErrorDocument": {
+    "Key": "reference/404.html"
+  },
+  "RoutingRules": [
+    {
+      "Condition": {
+        "KeyPrefixEquals": "reference/installation"
+      },
+      "Redirect": {
+        "HostName": "crystal-lang.org",
+        "HttpRedirectCode": "301",
+        "Protocol": "https",
+        "ReplaceKeyPrefixWith": "install"
+      }
+    },
+    {
+      "Condition": {
+        "KeyPrefixEquals": "reference/overview/hello_world.html"
+      },
+      "Redirect": {
+        "HostName": "crystal-lang.org",
+        "HttpRedirectCode": "301",
+        "Protocol": "https",
+        "ReplaceKeyPrefixWith": "reference/getting_started"
+      }
+    },
+    {
+      "Condition": {
+        "KeyPrefixEquals": "reference/overview"
+      },
+      "Redirect": {
+        "HostName": "crystal-lang.org",
+        "HttpRedirectCode": "301",
+        "Protocol": "https",
+        "ReplaceKeyPrefixWith": "reference/getting_started"
+      }
+    },
+    {
+      "Condition": {
+        "KeyPrefixEquals": "reference/${LATEST_VERSION}/${LATEST_VERSION}/"
+      },
+      "Redirect": {
+        "HostName": "crystal-lang.org",
+        "HttpRedirectCode": "301",
+        "Protocol": "https",
+        "ReplaceKeyWith": "404"
+      }
+    },
```

```suggestion
    {
      "Condition": {
        "KeyPrefixEquals": "reference/latest/${LATEST_VERSION}/"
      },
      "Redirect": {
        "HostName": "crystal-lang.org",
        "HttpRedirectCode": "302",
        "Protocol": "https",
        "ReplaceKeyWith": "reference/${LATEST_VERSION}/404.html"
      }
    },
```

This is the config that's currently doing the trick in production. I've also changed the redirect to a 302 so when/if we improve this to be an actual ErrorDocument (instead of a redirection) users will get it working better.

---

The actual chunk of JSON in production right now (so you can double-check) is this:

```    {
        "Condition": {
            "KeyPrefixEquals": "reference/latest/1.2/"
        },
        "Redirect": {
            "HostName": "crystal-lang.org",
            "HttpRedirectCode": "302",
            "Protocol": "https",
            "ReplaceKeyWith": "reference/1.2/404.html"
        }
    },```